frontend/src/components/Button.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    // your script goes here
+    export let btnState: boolean; 
+    export let onClickAction: any; 
```

Comment:
A justification of why onClickAction is type any would be great. A comment detailing why we're skipping type declaration is preferred.

Or if possible, set onClickAction to its expected type.

Reply:
> 

Ok. Thank you. Will look more into it.

Reply:
onClickAction is still of type any without justification.